Providers/Profile.swift

```diff
@@ -24,13 +21,26 @@ class ProfileFileAccessor : FileAccessor {
                 return nil
             }
         }
-
         return path
     }
 
-    func move(src: String, dest: String) -> Bool {
-        if let f = get(src) {
-            if let f2 = get(dest) {
+    func getDir(name: String?, basePath: String?) -> String? {
```

Comment:
I know this is just test cod but I think it is better style to have `func getDir(name: String?, basePath: String? = nil) -> String?` .. then you can leave out all the `basepath: nil` params where they are not used.